projects/schematics/src/migrations/5_0/constructor-deprecations/constructor-deprecations.ts

```diff
@@ -1,13 +1,15 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 import { migrateConstructorDeprecation } from '../../mechanism/constructor-deprecations/constructor-deprecations';
 import { CONSTRUCTOR_DEPRECATIONS_DATA } from './data/constructor-deprecations.migration';
+import { FORM_ERRORS_COMPONENT_MIGRATION } from './data/form-errors.component.migration';
+import { TAB_PARAGRAPH_CONTAINER_COMPONENT_MIGRATION } from './data/tab-paragraph-container.component.migration';
 
 export function migrate(): Rule {
   return (tree: Tree, context: SchematicContext) => {
-    return migrateConstructorDeprecation(
-      tree,
-      context,
-      CONSTRUCTOR_DEPRECATIONS_DATA
-    );
+    return migrateConstructorDeprecation(tree, context, [
```

Comment:
I'm assuming this was changed manually for the sake of fixing the conflict, and the breaking changes that were introduced recently, but I'm assuming we're going to re-generate this later instead of manually adding it to the array

Reply:
Or we can just add them in the breaking-change.json manually before regenerating

Reply:
same for other  deprecations, if applicable

Reply:
Let's discuss, not sure I understand.

Reply:
got it, but in short 
```suggestion
    return migrateConstructorDeprecation
    (
    tree, 
    context, 
    CONSTRUCTOR_DEPRECATIONS_DATA
    );
```

for every deprecation, we want it to all be included in their respective auto generated file. In this case, inside `CONSTRUCTOR_DEPRECATIONS_DATA`. 

I was asking if this was temporarily to fix the 'conflicts' from people merging deprecations still as this PR is only for 'rest' and not constructor deprecation